app/services/events/record.rb

```diff
@@ -343,6 +352,82 @@ def self.record_lead_provider_api_token_revoked_event!(author:, api_token:)
       new(event_type:, author:, heading:, lead_provider:, happened_at: Time.zone.now, metadata:).record_event!
     end
 
+    # Statement Adjustment Events
+
+    def self.record_statement_adjustment_added_event!(author:, statement_adjustment:)
```

Comment:
Does it make sense to pass all the values in here rather than traverse from `statement_adjustment`? The chances are we already have access to them in the controller and it generally makes testing easier. No strong feelings either way here though because there are only a couple being set.

Reply:
i think this keeps the controller clean, let the service class do all the work. adjustment knows all the information